book/eha.qmd

```diff
@@ -38,37 +40,50 @@ The *recurrent events setting* is as a special case of the multi-state setting b
+Mutually exclusive here means that once one of the events occurs, we can not observe the others (at the same time).
 As in the single event case, the events may not necessarily be terminal (such as death), instead they just signify the end of the period of interest for a given observation.
 
 Formally, let $Y$ be the time-to-event as before.
-In the competing risks setting, there is now an additional random variable $E_Y\in \{1,\ldots,k\}$, which denotes one of $k$ competing events that can occur at event time $Y$.
-The survival outcome notation, $(T, \Delta)$ has to be extended to accomodate for the possible competing risks.
-Previously, $\Delta = 1$ if right-censoring is observed, this can be modified by multiplying $\Delta$ by the index of the event that occurs.
-Now, $\Delta \in \{0, 1, \ldots, k\}$ where $\Delta = 0$ still indicates censoring and $\Delta = j$ means the $j$th event occurred.
+In the competing risks setting, there is now an additional random variable $E\in \{1,\ldots,k\}$ with realisations $e$, which denotes one of $k$ competing events that can occur at event time $Y$.
+The survival outcome notation, $(T, \Delta)$ has to be extended to accomodate for the possibility of competing risks.
+Previously, we defined the status indicator $\Delta \in \{0,1\}$ for right-censored data.
+In the competing risks setting we have $\Delta = I(Y_i \leq C_I) \wedge E_i = e$, such that
+$\Delta \in \{0, 1, \ldots, k\}$ where $\Delta = 0$ indicates censoring as before and $\Delta = e$ means the $e$th event occurred.
 
-The predominate approach to competing-risks analysis is known as *cause-specific hazards*, where $h_{0e}$ is defined as the hazard for the transition from initial state $0$ to state $e =1,\ldots,k$:
+A common approach to competing-risks analysis is known as *cause-specific hazards*, where $h_{e}$ is defined as the hazard for the transition from initial state $0$ to state $e =1,\ldots,k$:
 
 $$
-h_{0e}(\tau) = \lim_{\Delta \tau \to 0} \frac{P(\tau \leq Y \leq \tau + \Delta \tau, E = e\ |\ Y \geq \tau)}{\Delta \tau}, \; e = 1, \dots, k.
+h_{e}(\tau) = \lim_{\Delta \tau \to 0} \frac{P(\tau \leq Y \leq \tau + \Delta \tau, E = e\ |\ Y \geq \tau)}{\Delta \tau}, \; e = 1, \dots, k.
 $$
 
-As competing events are mutually exclusive at any time $\tau$, it is useful to further define the:
+As competing events are mutually exclusive at any time $\tau$, it is useful to further define the *all-cause hazard* which is the hazard of any event occuring:
 
-- *all-cause hazard*, $h(\tau) = \sum_{e = 1}^{k}h_{0e}(\tau)$: which is the...
-- *all-cause cumulative hazard*: $H(\tau) = \int_{0}^\tau h(u)\ du = \sum_{e=1}^{k}\int_{0}^\tau h_{0e}(u)\ du = \sum_{e=1}^k H_{0e}(\tau)$, and
-- *all-cause survival probability*: $S(\tau) = P(Y \geq \tau) = \exp(-H(\tau))$
+$$
+h(\tau) = \sum_{e = 1}^{k}h_{e}(\tau)
+$$
+
+the *all-cause cumulative hazard*
+
+$$
+H(\tau) = \int_{0}^\tau h(u)\ du = \sum_{e=1}^{k}\int_{0}^\tau h_{e}(u)\ du = \sum_{e=1}^k H_{e}(\tau)
```

Comment:
I'm a little confused. How do I calculate the all-cause cumulative hazard from the all-cause hazard? Is it just the integral of the all-cause hazard? Or is it the sum of individual cumulative hazards which are in turn the integral of the all-cause hzards?